pom.xml

```diff
@@ -135,9 +140,11 @@
         <handlebars.version>2.0.0</handlebars.version>
         <antlr.version>4.3</antlr.version>
         <lux.woodstox.version>4.1.1</lux.woodstox.version>
-        <commons-codec.version>1.10</commons-codec.version>
+        <commons-codec.version>1.11</commons-codec.version>
         <spring.version>4.3.10.RELEASE</spring.version>
         <jetty.version>9.3.14.v20161028</jetty.version>
+        <javax.annotation-api.version>1.3</javax.annotation-api.version>
+        <opensaml.osgi.version>3.3.0_2</opensaml.osgi.version>
```

Comment:
```xml
<javax.annotation-api.version>1.3</javax.annotation-api.version>
<opensaml.osgi.version>3.3.0_2</opensaml.osgi.version>
```
Do these 2 properties belong in the the `<!--START If these bundles are used elsewhere in DDF, the versions MUST exactly match the versions used by the CXF and Camel features.-->` section? If not, I don't think the version properties should be used in `features/pom.xml`:
```xml
<cxf.opensaml.osgi.version>1.3</cxf.opensaml.osgi.version>
<cxf.javax.servlet-api.version>3.3.0_2</cxf.javax.servlet-api.version>
```

Reply:
I'm not sure. I think I need to take a pass over the two big poms and clean up/clarify the properties.

Reply:
I'd still like to see this updated.

If these 2 version properties
```xml
<javax.annotation-api.version>1.3</javax.annotation-api.version>
<opensaml.osgi.version>3.3.0_2</opensaml.osgi.version>
```
are used in the cxf version properties
```xml
<cxf.javax.annotation-api.version>${javax.annotation-api.version}</cxf.javax.annotation-api.version>
<cxf.opensaml.osgi.version>${opensaml.osgi.version}</cxf.opensaml.osgi.version>
```
, they should be moved to the 
```xml
         <!--START If these bundles are used elsewhere in DDF, the versions MUST exactly match the versions used by the CXF and Camel features.-->
```
section. If they do not belong in that section, then the root pom version properties should not be used in `features/pom.xml`
 ```xml
<cxf.javax.annotation-api.version>1.3</cxf.javax.annotation-api.version>
<cxf.opensaml.osgi.version>3.3.0_2</cxf.opensaml.osgi.version>
```